$OpenBSD: patch-contrib_pdfmark_Makefile_sub,v 1.1 2012/01/01 21:05:33 schwarze Exp $
Fix parallel build race failure.
This is because the generated pdf files use the local generated
pdfroff helper script, but they don't depend directly upon it, so
make tries to create the two in parallel and randomly falls over.
Fix from upstream CVS by Mike Frysinger <vapier at gentoo dot org>.
--- contrib/pdfmark/Makefile.sub.orig	Fri Dec 31 08:33:09 2010
+++ contrib/pdfmark/Makefile.sub	Mon Dec 26 09:29:10 2011
@@ -1,4 +1,4 @@
-# Copyright (C) 2005, 2006, 2009 Free Software Foundation, Inc.
+# Copyright (C) 2005, 2006, 2009, 2011 Free Software Foundation, Inc.
 #      Written by Keith Marshall (keith.d.marshall@ntlworld.com)
 # 
 # This file is part of groff.
@@ -65,6 +65,9 @@ PDFROFF=\
 	$(PDFROFF) -mspdf --stylesheet=$(srcdir)/cover.ms $< >$@
 
 all: pdfroff $(make_pdfdoc)
+
+# The pdf files use the local script to generate.
+$(PDFDOCFILES): pdfroff
 
 pdfdoc: gnu.eps $(PDFDOCFILES)
 
